pallets/manta-sbt/src/lib.rs

```diff
@@ -650,6 +656,123 @@ pub mod pallet {
             });
             Ok(())
         }
+
+        #[pallet::call_index(9)]
+        #[pallet::weight(<T as pallet::Config>::WeightInfo::set_next_sbt_id())]
+        #[transactional]
+        pub fn set_next_sbt_id(
+            origin: OriginFor<T>,
+            asset_id: Option<StandardAssetId>,
+        ) -> DispatchResult {
+            ensure_root(origin)?;
+
+            NextSbtId::<T>::set(asset_id);
+            Self::deposit_event(Event::<T>::SetNextSbtId { asset_id });
+            Ok(())
+        }
+
+        #[pallet::call_index(10)]
+        #[pallet::weight(<T as pallet::Config>::WeightInfo::force_to_private())]
+        #[transactional]
+        #[allow(clippy::too_many_arguments)]
+        pub fn force_to_private(
+            origin: OriginFor<T>,
+            post: Box<TransferPost>,
+            mint_id: MintId,
+            metadata: BoundedVec<u8, T::SbtMetadataBound>,
+            minting_account: T::AccountId,
+        ) -> DispatchResultWithPostInfo {
+            let who = ensure_signed(origin)?;
+
+            let force_account = ForceAccount::<T>::get().ok_or(Error::<T>::NotForceAccount)?;
+            ensure!(who == force_account, Error::<T>::NotForceAccount);
+
+            let asset_id = id_from_field(post.asset_id.ok_or(Error::<T>::InvalidAssetId)?)
+                .ok_or(Error::<T>::InvalidAssetId)?;
+            Self::check_post_shape(&post, asset_id)?;
+            let sbt_metadata = MetadataV2::<T::SbtMetadataBound> {
+                mint_id,
+                collection_id: None,
+                item_id: None,
+                extra: Some(metadata),
+            };
+
+            // check that asset id is below max allowed number
+            ensure!(asset_id < MAX_FORCE_ASSET_ID, Error::<T>::TooHighAssetId);
```

Comment:
Can we make the max asset = the `NextSbtId` when called `setNextSbtId`.

Reply:
Yeah we could do that, my thought was just to set `NextSbtId` to MAX_FORCE_ASSET_ID . But we could make it `NextSbtId` instead might be cleaner.